iib/workers/tasks/build.py

```diff
@@ -1338,6 +1338,7 @@ def handle_regenerate_bundle_request(
     update_request(request_id, payload, exc_msg='Failed setting the bundle image on the request')
 
 
+@retry(wait_on=IIBError, logger=log)
```

Comment:
Is this too high level?

If the image cannot be pulled, the `podman create` command may fail. But it may also fail for completely unrelated reasons. 

What if right before running `podman create` we run `podman pull` inside this function? That will help us provide a better error message to the users. Wdyt?

Reply:
OK, availability of registry will be checked with pull then.